src/arviz_plots/plots/lm_plot.py

```diff
@@ -115,10 +119,30 @@ def plot_lm(
     visuals : mapping of {str : mapping or bool}, optional
         Valid keys are:
 
-        * ci_line -> passed to :func:`~.visuals.line_xy`. Defaults to False
-        * central_line -> passed to :func:`~.visuals.line_xy`.
-        * ci_fill -> passed to :func:`~.visuals.fill_between_y`.
-        * scatter -> passed to :func:`~.visuals.scatter_xy`.
+        * pe_line-> passed to :func:`~.visuals.line_xy`.
```

Comment:
Are spaces between lines and around `->` meaningful for doc building? Asking to make sure things are lay out as expected.